src/pages/course.js

```diff
@@ -32,6 +33,76 @@ const Course = props => {
     })
   }, [props.courseId])
 
+  const getFlattenedData = resData => {
```

Comment:
It's not clear to me why this is being done on the client, or even what exactly this is doing. Could you elaborate on the former, and (if this stays on the client) can you add some comments explaining what this does?

Also, it looks like this function doesn't close over any values in the component - this could be moved outside the component to keep the component logic itself clear.

Reply:
I can definitely move this to the api, that would make more sense.  As for what it does - from what I found, Sequelize doesn't have a way to not "nest" objects... So if I'm querying for a question and want to include info about the queue (like name, id) the question belongs to, the queue fields (name, id) will be nested under a larger "queue" object.  So this just flattens all of the questions returned.

Reply:
Makes sense - thanks for clarifying! This feels like one of those cases where Sequelize gets in our way much more than it helps us - raw SQL seems like it would be so much better here 🙂